app/assets/stylesheets/components/_sortable.scss

```diff
@@ -2,10 +2,18 @@
   cursor: move;
 }
 
+.sortable-list li {
+  padding-bottom: 2rem;
+  border-bottom: 1px solid govuk-colour("mid-grey");
+}
+
 .sortable-container {
   display: flex;
   justify-content: space-between;
 }
+.sortable-container div {
```

Comment:
I'm a little cautious about using `div` here as well, if we ever add more divs to any of the list items it'll affect those too. Perhaps give it a class name and use that as the selector?